src/js/components/skipnav.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const behavior = require('../utils/behavior');
+const once = require('receptor/once');
+
+const PREFIX = require('../config').prefix;
+
+const setTabindex = function (event) {
+  // NB: we know because of the selector we're delegating to below that the
+  // href already begins with '#'
+  const id = this.getAttribute('href').slice(1);
+  const target = document.getElementById(id);
+  if (target) {
+    target.setAttribute('tabindex', 0);
+    target.addEventListener('blur', once(event => {
+      target.setAttribute('tabindex', -1);
+    }));
+  } else {
+    // throw an error?
```

Comment:
Hrm, might want a single strategy for dealing cases where an expected element is missing. I see some other cases where an error is thrown: https://github.com/18F/web-design-standards/pull/1836/files#diff-6449513e1e61be79d366e960d8578ffbR16

I don't have a good feeling for what is "correct." On one hand, you would help developers by making problems obvious. On the other, it might break otherwise functional sites, which could be nasty for users.